src/home/CallList.tsx

```diff
@@ -17,11 +17,12 @@ limitations under the License.
 import { Link } from "react-router-dom";
 import { MatrixClient } from "matrix-js-sdk/src/client";
 import { RoomMember } from "matrix-js-sdk/src/models/room-member";
+import { Room } from "matrix-js-sdk";
```

Comment:
I think there have been issues in the past with importing the top-level `"matrix-js-sdk"` import from multiple places? Not sure if those still exist